### PR TITLE
[IE Tests] Added NormalizeL2 tests

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

File: inference-engine/tests/functional/plugin/shared/include/single_layer_tests/normalize_l2.hpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#include <vector>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+
+namespace LayerTestsDefinitions {
+
+using NormalizeL2LayerTestParams = std::tuple<
+        std::vector<int64_t>,               // axes
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/include/single_layer_tests/normalize_l2.hpp:18]: Please, align with [specification](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/normalization/LRN_1.md)

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/normalize_l2.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    InferenceEngine::Precision netPrecision;
+    std::tie(axes, eps, epsMode, inputShape, netPrecision, targetDevice) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(
+            ngraph::helpers::castOps2Nodes<ngraph::opset4::Parameter>(params));
+
+    auto normAxes = std::make_shared<ngraph::opset4::Constant>(ngraph::element::Type_t::i64, ngraph::Shape{axes.size()}, axes);
+
+    auto norm = std::make_shared<ngraph::opset4::NormalizeL2>(paramOuts[0], normAxes, eps, epsMode);
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/single_layer_tests/normalize_l2.cpp:43]: Looks like, we can create Normalize builder. Please move constant and norm node creation there

File: ngraph/core/reference/include/ngraph/runtime/reference/normalize_l2.hpp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            template <typename T>
+            void normalize_l2(const T* data,
+                              T* out,
+                              const Shape& data_shape,
+                              const AxisSet& reduction_axes,
+                              float eps,
+                              op::EpsMode eps_mode)
+            {
+                AxisSet axes = reduction_axes;
```
Comment (iefode) [inline ngraph/core/reference/include/ngraph/runtime/reference/normalize_l2.hpp:33]: @ilyachur @GlebKazantaev Could you please review NormalizeL2 reference implementation?

**Review by iefode — CHANGES_REQUESTED** [review]

**Review by iefode — APPROVED** [review]
Test part LGTM

File: ngraph/core/reference/include/ngraph/runtime/reference/normalize_l2.hpp (hunk 3 above)
Comment (iefode) [reply ngraph/core/reference/include/ngraph/runtime/reference/normalize_l2.hpp:33]: @ilyachur @GlebKazantaev ping

**Review by ilyachur — APPROVED** [review]

**Pull request merged**